lib/inspec/runner.rb

```diff
@@ -182,58 +182,10 @@ def register_rules(ctx)
 
     private
 
-    def block_source_info(block)
```

Comment:
I'll move it back.  The general reasons why I moved it was because with it here it makes are distinction between rspec runner a bit academic since this is tied directly to the rspec runner.  But this isn't particularly a hill I care about.